packages/email-templates/src/templates/keyMined.ts

```diff
@@ -20,5 +20,8 @@ export default {
 
 {{links txUrl openSeaUrl true}}
 
+{{#if transactionReceiptUrl}}
+<small><a href="{{transactionReceiptUrl}}">Transaction Receipt</a></small>
```

Comment:
Can you create a template for that so we DRY UP the code and use the same as above pleasE?